src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -437,9 +455,26 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
 
   /* Process the templated cluster init script and put all initialization files in the init bucket */
   private[service] def initializeBucketObjects(userEmail: WorkbenchEmail, googleProject: GoogleProject, clusterName: ClusterName, initBucketName: GcsBucketName, clusterRequest: ClusterRequest, serviceAccountKey: Option[ServiceAccountKey]): Future[Unit] = {
+
+
+    val jupyterNbExtensions = clusterRequest.userJupyterExtensionConfig match {
```

Comment:
Am wondering if this logic should go in `ClusterInitValues.apply` instead.

Also could make a generic function:
```
def templateMap[A, B](map: Map[A, B]): String = map.values.mkString(" ")
```